blocks/blockeligibilityvalidator.go

```diff
@@ -115,9 +116,12 @@ func (v BlockEligibilityValidator) BlockSignedAndEligible(block *types.Block) (b
 	vrfSig := block.EligibilityProof.Sig
 
 	if !v.validateVRF(vrfPubkey, message, vrfSig) {
+		v.log.Error("tortoise beacon eligibility VRF validation of beacon %v in epoch %v failed (counter: %v)", util.Bytes2Hex(epochBeacon), epochNumber, counter)
```

Comment:
make sure the calling function prints out the error, move this format to returned error and you can remove this log line